README.md

```diff
@@ -1,7 +1,8 @@
-# Simple Baselines for Human Pose Estimation and Tracking (sample)
-
+# Demo of Simple Baselines for Human Pose Estimation and Tracking
 This repository contains testing code for the paper https://arxiv.org/abs/1804.06208 .  <br>
```

Comment:
`testing`->`demo`

Reply:
for the paper [Simple Baselines for Human Pose Estimation and Tracking](https://arxiv.org/abs/1804.06208).